qiskit/providers/ibmq/api_v2/clients/websocket.py

```diff
@@ -140,6 +140,9 @@ def get_job_status(self, job_id, timeout=None):
         Args:
             job_id (str): id of the job.
             timeout (int): timeout, in seconds.
+            retry_max (int): max number of retries to
```

Comment:
I don't think the retry needs to be a parameter since we don't want to retry more than once anyway

Reply:
I thought having it specified as a parameter would allow for more flexibility in case we decide to retry more than once. But I agree, once should be enough 👍 . I'll go ahead an make the change.